docs/Debugging.md

```diff
@@ -0,0 +1,81 @@
+Debugging
+=========
+
+SGX-LKL provides a wrapper around gdb called `sgx-lkl-gdb`.
+This wrapper is compatible with IDEs and can be used instead of `gdb`.
+`sgx-lkl-gdb` automatically loads the SGX-LKL gdb plugin which
+ensures that debug symbols (if available) are loaded correctly. 
+When running in hardware mode, `sgx-lkl-gdb` uses the corresponding SGX debug
+instructions to read from and write to enclave memory.
+
+## Prerequisites
+
+### Open Enclave
+
+`sgx-lkl-gdb` relies on the Open Enclave gdb plugin.
+[Install Open Enclave](https://github.com/openenclave/openenclave/blob/master/docs/GettingStartedDocs/install_oe_sdk-Ubuntu_18.04.md) if necessary before continuing.
+
+The instructions below assume that Open Enclave was installed in the default directory `/opt/openenclave`.
+If you used a different directory, run `. <OE_PREFIX>/share/openenclave/openenclaverc` first.
```

Comment:
The statement implies we don't need to source `/opt/openenclave/share/openenclave/openenclaverc`.

Reply:
Yeah, that's the case. `sgx-lkl-gdb` checks the default directory. Should I mention that?